pkg/sql/sem/tree/name_resolution.go

```diff
@@ -576,7 +576,10 @@ type CommonLookupFlags struct {
 }
 
 // DatabaseLookupFlags is the flag struct suitable for GetDatabaseDesc().
-type DatabaseLookupFlags = CommonLookupFlags
+type DatabaseLookupFlags struct {
+	CommonLookupFlags
+	RequireMutable bool
```

Comment:
why isn't requireMutable part of commonLookupFlags at this point?

Reply:
That's a good point, I think it should be. I'll do that later.